frontend/src/components/intake/ShasIntakeForm.vue

```diff
@@ -302,9 +326,19 @@ onBeforeMount(async () => {
               :click-callback="clickCallback"
               title="Housing"
               icon="fa-house"
+              :class="{ 'p-red': validationErrors.includes('housing') }"
```

Comment:
Magic string. Should be referencing INTAKE_FORM_CATEGORIES. Same css class name comment as above.